temporian/core/operators/window/moving_max.py

```diff
@@ -31,6 +31,12 @@ def operator_def_key(cls) -> str:
         return "MOVING_MAX"
 
     def get_feature_dtype(self, feature: FeatureSchema) -> DType:
+        if not feature.dtype.is_numerical:
+            raise ValueError(
+                "moving_max requires numerical point inputs."
+                " Instead, get feature {feature.name!r} with type"
```

Comment:
s/get/got or received (same in the other messages)

Reply:
actually, would rephrase to "moving_max requires the EventSet to contain numerical features only, but received feature {} with type {}" (same in the other ones)